### PR TITLE
Fix Maths/IsDivisible

### PR DESCRIPTION
**Pull request by Masa-Shin**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/IsDivisible.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Checks if a number is divisible by another number.
+
+export const isDivisible = (num1, num2) => {
+  if (!Number.isFinite(num1) || !Number.isFinite(num2)) {
+    throw new Error('All parameters have to be numbers')
```
Comment (raklaptudirm) [inline Maths/IsDivisible.js:5]: As an enhancement, I would suggest throwing a [`TypeError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError).
```js
throw new TypeError("Expected a number")
```

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

**Comment by raklaptudirm** [comment]
@Masa-Shin I am pinging because it seems you have abandoned this PR.

**Comment by Masa-Shin** [comment]
@raklaptudirm 
I will fix it on this weekend thanks.

File: Maths/IsDivisible.js (hunk 1 above)
Comment (Masa-Shin) [reply Maths/IsDivisible.js]: @raklaptudirm 
Fixed!

**Review by raklaptudirm — APPROVED** [review]

**Comment by raklaptudirm** [comment]
Please fix the standard style issues

**Comment by raklaptudirm** [comment]
@Masa-Shin please fix the standard style issues.

**Comment by Masa-Shin** [comment]
@raklaptudirm 
sorry I fixed.

**Comment by raklaptudirm** [comment]
@Masa-Shin please fix the standard style issues

**Pull request merged**